server/cmd/mmctl/commands/user.go

```diff
@@ -336,6 +376,21 @@ Global Flags:
 {{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}
 `)
 
+	PreferenceListCmd.Flags().StringP("category", "c", "", "The optional category by which to filter")
+	PreferenceGetCmd.Flags().StringP("category", "c", "", "The category of the preference")
+	PreferenceGetCmd.Flags().StringP("name", "n", "", "The name of the preference")
+	_ = PreferenceGetCmd.MarkFlagRequired("name")
```

Comment:
Shouldn't `category` be required as well?

Reply:
Good catch, thank you!